RELEASE.md

```diff
@@ -98,11 +98,11 @@ To publish a release candidate:
 To publish a stable release:
 
 1. Do not change the release branch directly; make a PR to the release-X.Y branch with VERSION and any CHANGELOG changes.
-1. Ensure the `VERSION` file has **no** `-rc.X` suffix
-1. Update the Cortex version in the following locations:
-   - Kubernetes manifests located at `k8s/`
-   - Documentation located at `docs/`
-1. `git tag` the new release (see [How to tag a release](#how-to-tag-a-release))
+   1. Ensure the `VERSION` file has **no** `-rc.X` suffix
+   1. Update the Cortex version in the following locations:
+      - Kubernetes manifests located at `k8s/`
+      - Documentation located at `docs/`
```

Comment:
Was this intentional?

Reply:
Yes. I wanted to express that these points are part of first (make PR to the release branch...). PR should include these changes.